src/app/search/search.component.scss

```diff
@@ -152,3 +152,17 @@
   flex-wrap: wrap;
   justify-content: space-between;
 }
+
+.number-of-items {
+  position: absolute;
+  right: 30px;
```

Comment:
We try to avoid using pixels in styling. https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide. Use rems instead